interp/run.go

```diff
@@ -2213,6 +2213,51 @@ func compositeBinMap(n *node) {
 	}
 }
 
+func compositeBinSlice(n *node) {
+	value := valueGenerator(n, n.findex)
+	next := getExec(n.tnext)
+	child := n.child
+	if n.nleft == 1 {
+		child = n.child[1:]
+	}
+
+	values := make([]func(*frame) reflect.Value, len(child))
+	index := make([]int, len(child))
+	rtype := n.typ.rtype.Elem()
```

Comment:
Maybe we also need the part here to handle the cases for a slice of pointers, like in doCompositeBinStruct:

	if n.typ.cat == ptrT || n.typ.cat == aliasT {
		rtype = n.typ.val.rtype
	}

?